src/components/Glossary/GlossaryTooltip/GlossaryTooltip.stories.tsx

```diff
@@ -5,13 +5,21 @@ import GlossaryTooltipComponent from "."
 const meta = {
   title: "Molecules / Overlay Content / Glossary Tooltip",
   component: GlossaryTooltipComponent,
+  args: {
+    termKey: "big-endian",
+    children: "big-endian",
+  },
 } satisfies Meta<typeof GlossaryTooltipComponent>
 
 export default meta
 
-export const GlossaryTooltip: StoryObj<typeof meta> = {
+type Story = StoryObj<typeof meta>
+
+export const Basic: Story = {}
+
+// for chromatic story snapshot showing the rendered popover
+export const OnOpen: Story = {
```

Comment:
Need to add a decorator here, where it provides enough white-space to cover the visible tooltip. Otherwise the tooltip will be cropped out in the chromatic snapshot.

The decorator can be supplied at the individual story level. (There is currently a decorator at the component level for the [Tooltip](https://github.com/ethereum/ethereum-org-website/blob/dev/src%2Fcomponents%2FTooltip%2FTooltip.stories.tsx#L46-L52) for this very reason)